spec/services/idempotency/conflict_error_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Idempotency::ConflictError do
+  it { expect(described_class).to be < StandardError }
```

Comment:
```suggestion
  it { expect(described_class).to be_a(StandardError) }
```

Reply:
Hi @willfish unfortunately that doesn't work: the current test is checking that it inherits from StandardError, not that it is a StandardError. This is the output with the suggested change:
```
   Failure/Error: it { expect(described_class).to be_a(StandardError) }
       expected Idempotency::ConflictError to be a kind of StandardError
```